data-pipeline/src/data_pipeline/pipelines/gnomad_v4_variants.py

```diff
@@ -38,63 +39,76 @@
 
 config = PipelineConfig(
     name=pipeline_name,
-    input_root="gs://gnomad-v4-data-pipeline/input",
-    output_root="gs://gnomad-v4-data-pipeline/output",
+    #   input_root="gs://gnomad-v4-data-pipeline/input",
+    #   output_root="gs://gnomad-v4-data-pipeline/output",
+    input_root="gs://gnomad-browser-data-pipeline/phil-scratch/input",
+    output_root="gs://gnomad-browser-data-pipeline/phil-scratch/output",
 )
 
 
 pipeline = Pipeline(config=config)
 
+#   pipeline.add_task(
+#       name="prepare_gnomad_v4_variants",
+#       task_function=prepare_gnomad_v4_variants,
+#       output_path=f"{output_sub_dir}/gnomad_v4_variants_base.ht",
+#       inputs={
+#           "exome_variants_path": "gs://gcp-public-data--gnomad/release/4.1/ht/exomes/gnomad.exomes.v4.1.sites.ht",
+#           "genome_variants_path": "gs://gcp-public-data--gnomad/release/4.1/ht/genomes/gnomad.genomes.v4.1.sites.ht",
+#           "variants_joint_frequency_path": "gs://gcp-public-data--gnomad/release/4.1/ht/joint/gnomad.joint.v4.1.sites.ht",
+#       },
+#   )
```

Comment:
Since this pipeline has been run with the new task, can these steps be un-commented again?

I imagine that the purpose of commenting these out was to ensure that these steps didn't run, so that you could take the old output and annotate on top of that. It appears as though that has happened. As such, could we return the pipeline to the previous, un-commented state, now with the inclusion of your additional step?

Reply:
indeed, I meant to do so but apparently either forgot or accidentally reverted it